### ISSUE TITLE
Allow admins to add unique links and remove existing ones on the live site

### ISSUE THREAD

**Issue by yousefa00** [hack4impact-uiuc/link-shortener#2]
Rather than submitting new commits for every new shortened link, admins should be able to sign-in at h4i.app and add/remove forwards.

**Comment by arpanlaha** [comment]
IMO this requires too much work to be worth it, as we'd need a full stack app with auth and a database - I don't think the benefit outweighs the development time cost, and its not a very interesting technical problem either in my eyes (I wouldn't want to spend time writing it out as I don't find it compelling engineering work).

**Comment by yousefa00** [comment]
Yeah, not sure why you were assigned for this. I'll work on it this summer, or have the infra team work on it. Will also be rewriting the backend completely in express and hooking into the memberDB tool's api to figure out which members are actually registered in our org (which i think is a pretty cool use for it).

**Comment by arpanlaha** [comment]
Honestly I still don't think it's that worth it as we do have our own membership list through GitHub - I only see significant value in having this be someone's first full stack app as it sits in the weird spot of extremely simple business logic that still requires a full stack which is too much work for how simple it is. Up to you all ofc on final call though

**Comment by yousefa00** [comment]
Appreciate the note, but making commits for every change is not the way I want this infra to work. I also don't think authentication and the backend are a lot of work, as it's nothing new from what we've done with memberDB tool, and think this would be a good project for the summer with its small scope.

**Comment by arpanlaha** [comment]
If there are features I would want to see from a more complex version it would be link visibility - which shortcuts do we want put on the main page vs. which we don't (like an "unlisted" link)

**Comment by arpanlaha** [comment]
Additionally maybe analytics of some sort
